src/Yoti.Auth/Images/Image.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Yoti.Auth.Images
+{
+    public abstract class Image
+    {
+        private readonly AttrpubapiV1.ContentType _type;
```

Comment:
Concrete subclasses will know the Type of Image they represent, so this _Type_ property should not be required.

And we'd be better off removing references to any protobuf generated types.  Try to keep them contained in the converters - don't let the dependency spread around if possible, even if we don't expose it.